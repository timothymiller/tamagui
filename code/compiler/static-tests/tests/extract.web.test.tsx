process.env.IDENTIFY_TAGS = 'true'

import './extract-tests'
