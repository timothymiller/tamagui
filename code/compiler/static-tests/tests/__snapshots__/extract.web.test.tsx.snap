// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`extract-tests > converts a style object to class names 1`] = `":root ._bg-red{background-color:red;}"`;

exports[`extract-tests > converts a style object to class names 2`] = `":root ._transform-rotateY10de41674{transform:rotateY(10deg);}"`;

exports[`extract-tests > converts a style object to class names 3`] = `":root ._bxsh-0px0px10pxr896772428{box-shadow:0px 0px 10px rgba(255,0,0,1);}"`;

exports[`extract-tests > supports RTL properties 1`] = `
[
  [
    "paddingStart",
    "100px",
    "_paddingStart-100px",
    undefined,
    [
      ":root ._paddingStart-100px{padding-start:100px;}",
    ],
  ],
  [
    "paddingEnd",
    "10px",
    "_paddingEnd-10px",
    undefined,
    [
      ":root ._paddingEnd-10px{padding-end:10px;}",
    ],
  ],
  [
    "marginStart",
    "50px",
    "_marginStart-50px",
    undefined,
    [
      ":root ._marginStart-50px{margin-start:50px;}",
    ],
  ],
  [
    "marginEnd",
    "2px",
    "_marginEnd-2px",
    undefined,
    [
      ":root ._marginEnd-2px{margin-end:2px;}",
    ],
  ],
]
`;
